examples/design-system-tutorial/src/main.ts

```diff
@@ -0,0 +1,132 @@
+import {
+    accentPalette,
+    baseLayerLuminance,
+    ColorRecipe,
+    controlCornerRadius,
+    density,
+    fillColor,
+    fluentButton,
+    fluentCard,
+    fluentTreeItem,
+    fluentTreeView,
+    neutralFillLayerRecipe,
+    neutralLayer1,
+    neutralPalette,
+    PaletteRGB,
+    provideFluentDesignSystem,
+    StandardLuminance,
+    SwatchRGB,
+} from "@fluentui/web-components";
+import { parseColorHexRGB } from "@microsoft/fast-colors";
+import { html } from "@microsoft/fast-element";
+import { demoCard, fluentAttribution, Navigation } from "./components";
+
+Navigation; // Prevent tree shaking.
```

Comment:
Should we add additional comments here as to *why* this exists? IE, because it uses the decorator syntax, we need to create a hard reference to the class in the file to ensure it's not tree shaken away. That isn't necessary for the others due to....